segmentation_labeling_app/rois/rois.py

```diff
@@ -24,10 +25,112 @@
 sql_insert_roi = """ INSERT INTO rois_prelabeling (transform_hash,
                      ophys_segmentation_commit_hash,
                      creation_date, upload_date, manifest, experiment_id,
-                     roi_id) 
+                     roi_id)
                      VALUES (?, ?, ?, ?, ?, ?, ?) """
 
 
+def cumulative_fraction_threshold(data, target_fraction, nbins=200):
```

Comment:
Is nbins not used in this function? It's also not in the docstring.

Reply:
right. left over from a previous attempt. removing.